mainmatter/introduction/graphics_apis/index.tex

```diff
@@ -1,33 +1,108 @@
+In addition, the high-level \glspl{API} most often use the low-level \glspl{API} to do the heavy lifting.
+Therefore, the developments at the lower level have wide spread consequences.
+Today the development of graphics \glspl{API} and the underlying driver libraries is done be separate companies.
+Often one company designs the \gls{API}, and hardware manufacturers can then decide to support the \gls{API} on their \glspl{GPU}.
+To add support for an \gls{API} on a specific \gls{GPU}, the manufacturer includes a library implementation as part of the device driver.
+Usually the driver must pass some tests and requirements, for the \gls{API} to be officially supported on the device.   
+
+The most popular low-level graphics \glspl{API} during the last game console generation, lasting from 2005 to about 2017, have been Microsoft’s Direct3D, part of the DirectX multimedia API collection, and Khronos’ OpenGL. 
+Direct3D is designed by Microsoft to be used exclusively on the Microsoft Windows platform. 
+The underlying driver is partially implemented by Microsoft in a closed common runtime, which communicates with the hardware vendors actual \gls{GPU} driver through the Direct3D \gls{DDI}\cite{dxDDI}.
+On the other hand, OpenGL is designed by several hardware and software vendors in coorperation, and requires that manufacturers write their own library from scratch. 
+
+A benefit of OpenGL is that manufacturers may provide extensions to the \gls{API} to give developers access to new hardware features \cite{openGLExtensions}.
+With Direct3D, new hardware features are only provided to developers, when a new version of the \gls{API} is released by Microsoft. 
+One of the big differences between the two \glspl{API} is the extent of their platform support.
+Direct3D 11, being one of the latest releases, is supported on PCs running the Windows \gls{OS}, smartphones running the Windows Phone \gls{OS} as well as on Microsoft’s Xbox One.
+OpenGL, with OpenGL 4 being the latest major release, is considered cross platform.
+If a driver has been written, there is support for it on both Windows and Unix-based systems like macOS, iOS, Linux and Android. 
+
+Getting a clear picture of the market share of the two \glspl{API} is rather difficult.
+Though it seems that for many years Direct3D has had and upper hand.
+The people behind the popular Unity game engine collect system specifications on Unity users, as to help developers figure out what system configurations their games should support.
+Looking at the statistics from March 2017, only about 1\% of users use a non-Windows \gls{OS} \cite{unityStats}.
+Similar statistics are collected by Valve Corporation from users of their digital distribution platform, Steam.
+This shows a similar 3\% statistic for non-windows users\cite{steamStats}, meaning that a large portion of gamers play on systems with some version of Direct3D enabled.
```

Comment:
I wouldn't say meaning that a large portion of gamers use Direct3D, wouldn't assumption be better? Steam and Unity doesn't collect data on everyone afterall.